resume.html

```diff
@@ -11,80 +11,89 @@ <h2>Hey!</h2>
   <hr>
   <div class="section">
     <h1>Experience</h1>
+    <h2>Web Developer</h2>
+    <h3>Lake Monster Brewing Company</h3>
+    <h4>Saint Paul, MN | October 2017 — Present</h4>
+    <div>
+      <p>Completed a website makeover that improved readability, accessibility, and asthetic appeal, including the responsive food truck schedule widget that I built with <a href="https://twitter.com/alexeives">@alexeives</a>. Kept the website taplist and events up-to-date, then upgraded the taplist to update automagically with the Untappd API.</p>
+      <li><strong>Stack:</strong> SquareSpace CMS, HTML/CSS, JavaScript</li>
+      <li><strong>Tools:</strong> GitLab</li>
+    </div>
 
-    <h2>Web Developer / Owner, Very Good Websites Co.</h2>
-    <h3>Saint Paul, MN | July, 2019 — Present </h3>
     <div>
-      <p>Custom website development and individualized technology help for my clients.</p>
+      <h2 class="agency-section">TEKsystems Inc. Assignments</h2>
+      <div class="agency-roles">
+        <h2>Full Stack Development, Q3C / Primoris</h2>
+        <h4>Remote | March — July, 2020</h4>
+        <p>Coached the team around Agile strategies, and implemented practices to improve team communication and cohesion. Collaborated with other developers on debugging tools, bug fixes. Mentored newer developer.</p>
+        <li><strong>Stack: </strong>Angular, TypeScript, Node, MSSQL</li>
```

Comment:
Consider re-ordering these
```suggestion
        <li><strong>Stack: </strong>TypeScript, Node, Angular, MSSQL</li>
```